commcare/src/main/java/org/motechproject/commcare/web/CommcareController.java

```diff
@@ -56,7 +58,12 @@ public String handleMiscException(Exception e) {
 
     private String handleException(Exception e) {
         logger.error(e.getMessage(), e);
-        return e.getMessage();
+
+        JsonObject jsonObject = new JsonObject();
+        jsonObject.addProperty("message", e.getMessage());
+
+        Gson gson = new Gson();
+        return gson.toJson(jsonObject);
```

Comment:
We should make change only in frontend, this type of handling exception in backend is bad